srcpkgs/gtk-layer-shell/template

```diff
@@ -1,9 +1,10 @@
 # Template file for 'gtk-layer-shell'
 pkgname=gtk-layer-shell
 version=0.5.1
-revision=1
+revision=2
 build_style=meson
 build_helper="gir"
+configure_args="-Dtests=true"
```

Comment:
cc @travankor

Reply:
```bash
if [ "$XBPS_CHECK_PKGS" ]; then
	configure_args+=" -Dtests=true"
fi
```
is probably a better bet, so we only enable tests when somebody is actually running them.

Reply:
Good call, pushed your suggestion